client/components/date-control/index.tsx

```diff
@@ -18,6 +18,7 @@ const DateControl = ( {
 	dateRange,
 	overlay,
 	shortcutList,
+	use40pxDefaultSize = false,
```

Comment:
Were these meant for a separate PR? In any case, given the low usage of these components, I feel like it'd be better if we just upsized all of the instances instead of exposing a new prop.